wiki/Twitter/de.md

```diff
@@ -0,0 +1,24 @@
+# Twitter
+
+osu! hat mehrere Twitter-Konten, die alle eine bestimmte Funktion erfüllen. Auf den meisten ist nur wenig los, aber sie helfen dir dabei, über osu! informiert zu bleiben.
+
+## Service
+
+| Avatar | Kürzel | Beschreibung |
+| :-: | :-: | :-- |
+| ![osu!status Avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Benachrichtigungen von Fehlern auf der Website und von [Bancho](/wiki/Bancho_(server)). |
+| ![osu! Supportavatar](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Hilfe bei Fragen über den Account und/oder Problemen in der Community. Wird von dem [Account Supportteam](/wiki/People/The_Team/Account_support_team) betrieben. |
+
+## Community
+
+| Avatar | Kürzel | Beschreibung |
+| :-: | :-: | :-- |
+| ![osu! Avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Die offizielle Quelle für Nachrichten und Ankündigungen. |
+| ![BanchoBoat Avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Gedacht für einen spaßigen Austausch, falls Dinge schief laufen. |
```

Comment:
```suggestion
| ![BanchoBoat Avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Gedacht für einen spaßigen Austausch, wenn Dinge schief laufen. |
```

Reply:
In my opinion, `falls` is for cases, where the future is uncertain. `Wenn` is used, when something is planned but it is not sure when it will happen. Server outages and other issues are unplanned situations, so I would suggest to use `falls` here.

Reply:
- Anything that can go wrong will  go wrong :)
- As I understand it you could say the same thing for `if` and `when`, and the original page uses when.

Not to fussed either way. I leave it up to you.